content/docs/command-reference/add.md

```diff
@@ -336,91 +339,78 @@ $ tree .dvc/cache
 Only the hash values of the `dir/` directory (with `.dir` file extension) and
 `file2` have been cached.
 
-## Example: Transfer to the cache
+## Example: Caching large data externally {#straight-to-cache}
 
-When you have a large dataset in an external location, you may want to add it to
-the <abbr>project</abbr> without having to copy it into the workspace. Maybe
-your local disk doesn't have enough space, but you have setup an
-[external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
-that could handle it.
+Sometimes you may want to add a large dataset currently found in an external
+location, so it becomes local to the project. But what if there's not enough
```

Comment:
`so it becomes local to the project` - it will be hard to understand to be honest? can we make somehow explicit?

Reply:
Agreed. I removed "so it becomes local to the project". The next sentence ("But what if there's not enough disk space to download") implies that we mean to add locally.